airflow/providers/google/cloud/operators/bigquery.py

```diff
@@ -2109,3 +2122,287 @@ def execute(self, context: Any):
+        :type bigquery_conn_id: str
+        :param bq_dataset_name:  The name of the BigQuery data set where any temp tables will be created that are needed
+            as part of the GE validation process.
+        :type bq_dataset_name: str
+        :param send_alert_email:  Send an alert email if one or more expectations fail to be met?  Defaults to True.
+        :type send_alert_email: boolean
+        :param datadocs_link_in_email:  Include in the alert email a link to the data doc in GCS that shows the
+            validation results?  Defaults to False because there's extra setup needed to serve HTML data docs stored in
+            GCS.  When set to False, only a GCS path to the results are included in the email.
+            Set up a GAE app to serve the data docs if you want a clickable link for the data doc to be included in the
+            email.  See here for set up instructions:
+            https://docs.greatexpectations.io/en/latest/guides/how_to_guides/configuring_data_docs/how_to_host_and_share_data_docs_on_gcs.html
+        :type datadocs_link_in_email: boolean
+        :param datadocs_domain: The domain from which the data docs are set up to be served (e.g. ge-data-docs-dot-my-gcp-project.ue.r.appspot.com).
+            This only needs to be set if datadocs_link_in_email is set to True.
+        :type datadocs_domain: str
+        :param email_to:  Email address to receive any alerts when expectations are not met.
+        :type email_to: str
+        :param fail_if_expectations_not_met: Fail the Airflow task if expectations are not met?  Defaults to True.
+        :type fail_if_expectations_not_met: boolean
+    """
+
+    _EMAIL_CONTENT = '''
+            <html>
+              <head>
+                <meta charset="utf-8">
+              </head>
+              <body style="background-color: #fafafa; font-family: Roboto, sans-serif=;">
+                <div style="width: 600px; margin:0 auto;">
+                    <div style="background-color: white; border-top: 4px solid #22a667; border-left: 1px solid #eee; border-right: 1px solid #eee; border-radius: 6px 6px 0 0; height: 24px;"></div>
+                        <div style="background-color: white; border-left: 1px solid #eee; border-right: 1px solid #eee; padding: 0 24px; overflow: hidden;">
+                          <div style="margin-left: 35px;">
+                            Great Expectations Alert<br>
+                            One or more data expectations were not met in the {0} file. {1}
+                       </div>
+              </body>
+            </html>
+            '''
+
+    @apply_defaults
+    def __init__(self, *, gcp_project, expectations_file_name, gcs_bucket, gcs_expectations_prefix,
+                 gcs_validations_prefix, gcs_datadocs_prefix, validation_type, validation_type_input,
+                 bq_dataset_name, email_to, datadocs_domain='none', send_alert_email=True, datadocs_link_in_email=False,
```

Comment:
@turbaszek , as suggested, made the datadocs domain a parameter instead of setting it from an Airflow variable.